etc/cmake/options.cmake

```
@@ -132,6 +132,11 @@ if(OT_CSL_DEBUG)
     target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE=1")
 endif()
 
+option(OT_SSED_SSED_LINK "enable SSED-to-SSED link")
```

```suggestion
option(OT_SSED_LINK "enable SSED-to-SSED link")
```

`SSED_LINK` is clear enough?

---

`SSED_LINK` could mean SSED to a Thread Router. From Spec, it is called SSED-TO-SSED link.